04-understanding-survey-data-documentation.Rmd

```diff
@@ -213,14 +202,14 @@ cps_state %>%
 targetpop <- cps_state %>%
   as_tibble() %>%
   filter(PRTAGE >= 18,
-         PRCITSHP %in% (1:4)) %>%
+         PRCITSHP %in% (1:4)) %>% 
   pull(PWSSWGT) %>%
   sum()
 
 targetpop
 ```
 
-The target population in 2020 is `r scales::comma(targetpop)`. This information gives us what we need to create the post-election survey object with {srvyr}.
+The target population in 2020 is `r scales::comma(targetpop)`. This information gives us what we need to create the post-election survey object with {srvyr}. 
```

Comment:
We should explain a bit more about what the target population is and walk through each of the analysis steps to get there.  OR we could just go to the CPS page and pull up the # and just cite it.  May be eaiser?